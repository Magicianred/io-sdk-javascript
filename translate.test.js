const importer = require("./importer")
const translate = require("./translate")

test("translate", () => {
    let imp = importer("data/test1.xlsx")
    //console.log(imp)
    let tran = translate(imp[0])
    //console.log(tran)
    expect(tran).toMatchSnapshot()
})