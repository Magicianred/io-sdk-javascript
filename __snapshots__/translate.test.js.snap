// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`translate 1`] = `
Object {
  "content": Object {
    "due_date": 2001-01-01T00:00:00.000Z,
    "markdown": "# Wecome, Giovanni Rossi\\\\nThis is a test message.\\\\n.This message was sent using the IO-SDK.\\\\nYour fiscal code is ISPXNB32R82Y766F\\\\nWe hope you are enjoing using IO.\\\\n",
    "subject": "Hi Giovanni",
  },
  "fiscal_code": "ISPXNB32R82Y766F",
  "time_to_live": 3600,
}
`;
